jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     '^.+\\.(ts|tsx|js|jsx)$': require.resolve('ts-jest'),
   },
   transformIgnorePatterns: [
-    'node_modules/(?!(ol)/)', // <- exclude the open layers library
+    'node_modules/(?!(ol)|(monaco-promql)/)', // <- exclude the open layers library
```

Comment:
When we don't have this added to jest we have the following error:
```
https://stackoverflow.com/a/49676319/767660

/Users/ismailsimsek/Documents/workspace/grafana/.yarn/cache/monaco-promql-npm-1.7.4-1b0c13a666-9fe2708be5.zip/node_modules/monaco-promql/index.js:23
  export * from './promql/promql.contribution';
  ^^^^^^
  
  SyntaxError: Unexpected token 'export'
```

Reply:
Instead of this, I would just mock Monaco Editor as we do in other tests (e.g. https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/components/PromQueryEditorByApp.test.tsx#L13)

```
jest.mock('../../components/monaco-query-field/MonacoQueryFieldWrapper', () => {
  const fakeQueryField = () => <div>prometheus query field</div>;
  return {
    MonacoQueryFieldWrapper: fakeQueryField,
  };
});
```

Reply:
Thanks for that! I will update the file.